python-frontend/src/test/java/org/sonar/python/tree/PythonTreeMakerTest.java

```diff
@@ -1195,6 +1195,11 @@ public void assignement_expression() {
 
     assertThat(assignmentExpression.children()).containsExactly(name, walrus, walrusExpression);
 
+    setRootRule(PythonGrammar.EXPR);
+    astNode = p.parse("foo(a:=42)");
+    expression = treeMaker.expression(astNode);
+    assertThat(expression.is(Kind.CALL_EXPR)).isTrue();
+
```

Comment:
I think we could add an assertion regarding the first argument being an assignment expression.